src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -1283,7 +1283,7 @@ private void Hide(object parameters)
 
             if (this.IsHomeSpace)
             {
-                if (DynamoViewModel.CloseHomeWorkspaceCommand.CanExecute(null))
+                if (DynamoViewModel.CloseHomeWorkspaceCommand.CanExecute(true))
```

Comment:
I guess by reading the code, instead of forcing a true to overwrite the function, you can simply remove this check? Seems equivalent

Reply:
yes it is the same, will remove it. I was thinking, may be as a good practice, 'CanExcute' is checked before the command is executed.

Reply:
See my comments above..